CMakeLists.txt

```diff
@@ -112,6 +112,11 @@ target_compile_definitions(${PROJECT_NAME}
     $<$<BOOL:${RCUTILS_AVOID_DYNAMIC_ALLOCATION}>:RCUTILS_AVOID_DYNAMIC_ALLOCATION>
   )
 
+configure_file(
```

Comment:
Why do you keep the `target_compile_definitions`?
Why there is not `RCUTILS_NO_THREAD_SUPPORT` in *configuration_flags.h*?

Reply:
Done